### ISSUE TITLE
Vim syntax highlight support on .tmux.local.conf file

### ISSUE THREAD

**Issue by perseoGI** [gpakosz/.tmux#469]
Hey! To improve readability on vim/nvim could you please include following line at the button of .tmux.local.conf?

`# vim:syntax=tmux
`

This will enable tmux default syntax highlighting for local tmux config file.

Thanks :3

**Comment by gpakosz** [comment]
Hello @perseoGI 👋

I will consider it. I noticed that Vim used to apply proper syntax highlighting until recently. I wonder what changed on Vim's side 🤔

**Comment by perseoGI** [comment]
Hmm I think you are right! 
Vim does highlight the local file without that code line. 
But Neovim need that line to highlight. 

So this issue should be named as "Neovim support syntax highlight" ^^
Thanks

**Comment by gpakosz** [comment]
Hey @perseoGI 👋

So your suggestion should be 
```
# vim: set syntax=tmux
```

And it only works if Vim's `modeline` is enabled, which many disable for security measures.

What Vim plugin are you using for tmux syntax highlight btw?

**Comment by perseoGI** [comment]
Yes... recently I saw `modeline` is being disabled by many users, it supposes a huge security breach.

Hmm that is really a good question... it seems that when starting neovim with -u NONE (and setting syntax on) it is detecting `.tmux.local.conf` as a configuration filetype (doing a more less good job).

After some testing on my vim config I found the plugin which was disabling syntax highlight on this file: `sheerun/vim-polyglot`
This is may be a good time to uninstall this plugin (there are now better alternatives)

But I must admit something, even without `polyglot`, the syntax highlight is still "poor", it does not detect tmux typical keybindings (this happens either with vim and neovim). 
My recommendation is to add that mentioned line from yesterday to the end of the file. 

If a user has set `modelines=0`, it will not notice anything. But for the ones who haven't disable this option, a better experience will get :)
